galaxy/accounts/tests/test_custom_user_model.py

```diff
@@ -0,0 +1,374 @@
+# -*- coding: utf-8 -*-
```

Comment:
Missing copyright string as a part of Apache-2.0 license header.

Reply:
Yes, and I have a question.
I'm not Red Hat employee. Should I put Red Hat copyright in headers — `(c) 2012-2018, Ansible by Red Hat`? Or may I just write `(c) 2012-2018, Ansible`?

Reply:
I will fix it, but I'm not sure what shall I write here.

Reply:
Leave it as is for now. We need to clarify our policies regarding copyrights.

Reply:
@cutwater So for this and other PRs from my side I will add banners **without**
```
# (c) 2012-2018, <whatever>
``` 
line. Correct?

Reply:
Yes, we can fix it later.